README.adoc

```diff
@@ -14,8 +14,8 @@ http://www.gradle.org/[Gradle] is managed as another tool inside Jenkins (the sa
 
 It also allows detecting https://gradle.com/build-scans/[Build Scans] in arbitrary console logs, for Maven and Gradle builds and display them in the Jenkins UI.
```

Comment:
```suggestion
It also allows detecting https://gradle.com/build-scans/[Build Scan] links in arbitrary console logs, for Maven and Gradle builds and display them in the Jenkins UI.
```

Can you check and rename all occurrences of `Build Scans` to `Build Scan`? If we still need the plural version, do not write it in upper-case.

Reply:
Done